perf/reader.go

```diff
@@ -16,8 +17,9 @@ import (
 )
 
 var (
-	ErrClosed = os.ErrClosed
-	errEOR    = errors.New("end of ring")
+	ErrClosed  = os.ErrClosed
+	ErrTimeOut = os.ErrDeadlineExceeded
```

Comment:
This alias isn't necessary. Just document that `os.ErrDeadlineExceeded` is returned.